code/modules/mob/living/simple_animal/distortion/legend/papa_bongy.dm

```diff
@@ -0,0 +1,332 @@
+	..()
+	var/units_to_add = list(
+		/mob/living/simple_animal/hostile/bongy_hostile = 1,
+		)
+	AddComponent(/datum/component/ai_leadership, units_to_add, 8, TRUE, TRUE)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+//Unmanifesting is not linked to any proc by default, if you want it to happen during gameplay, it must be called manually.
+/mob/living/simple_animal/hostile/distortion/papa_bongy/attacked_by(obj/item/I, mob/living/user)
+	. = ..()
+	if(istype(I, /obj/item/food/fried_chicken))//Fried chicken will require certain condiments to de-manifest in the future. For now, keep it simple
+		for(var/goop in I.reagents.reagent_list)
+			var/datum/reagent/R = goop
+			if(istype(R, /datum/reagent/consumable/capsaicin))
+				if(R.volume >= 40)//is it REALLY spicy?
+					say("BUGAAAAAAAAAAAAAAAAAAAAAAAAWK!")
+					to_chat(user, "[src] grimaces, sputters and coughs! It begins thrashing about, and you feel the temperature around you rise...")
+					new /mob/living/simple_animal/hostile/distortion/papa_bongy/spicy(get_turf(src))
+					qdel(src)
+					break
+			if(R.type in yucky_list)
+				say("BAWK BAWK!!!")
+				to_chat(user, "[src] didn't like that at all.")
+				adjustBruteLoss(150)
+				break
+			happy = TRUE
+	if(istype(I, /obj/item/food/pizzaslice))
+		if(I.type in bad_pizza_list)
+			say("BAWK BAWK!!!")
+			to_chat(user, "[src] didn't like that at all.")
+			adjustBruteLoss(150)
+			qdel(I)
+			return
+		happy = TRUE
+	if(happy == FALSE)
+		qdel(I)
+		return
+	else
+		qdel(I)
+		can_act = FALSE
+		addtimer(CALLBACK(src, PROC_REF(Unmanifest)),3 SECONDS)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/OpenFire(atom/A)
+	if(!can_act)
+		return FALSE
+	if(get_dist(src, A) <= 2) //no shooty in mele
+		return FALSE
+	if(LAZYLEN(unit_list) >= units_max)
+		return FALSE
+	ThrowBongy(A)
+	return ..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/Move()
+	if(!can_act)
+		return FALSE
+	..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/CanAttack(atom/the_target)
+	if(ishuman(the_target))
+		var/mob/living/carbon/human/L = the_target
+		if(L.sanity_lost && L.stat != DEAD)
+			if(HAS_AI_CONTROLLER_TYPE(L, /datum/ai_controller/insane/murder/bongy))
+				return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/AttackingTarget()
+	if(!can_act || !target)
+		return FALSE
+	if(!isliving(target))
+		return ..()
+	var/mob/living/carbon/human/H = target
+	if(ishuman(H))
+		if(istype(target.ai_controller, /datum/ai_controller/insane/murder/bongy))
+			LoseTarget()
+			return//need to test whether this is still needed
+		H.add_movespeed_modifier(/datum/movespeed_modifier/clowned)
+		addtimer(CALLBACK(H, TYPE_PROC_REF(/mob, remove_movespeed_modifier), /datum/movespeed_modifier/clowned), 2 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
```

Comment:
Why does this use a random movespeed modifier? You should make your own so that it isn't confusing to anyone reading variables.